File: modul 5/PRAK501-2410817320004-AmandhaCitraMustika.py
def MaxBilangan(a, b, c, d):
    if a>b and a>c and a>d :
        return a
    elif b>a and b>c and b>d :
        return b
    elif c>a and c>b and c>d :
        return c
    else :
        return d
    
a, b, c, d = map(int, input().split())
hasil = MaxBilangan(a, b, c, d)
print(f"{hasil}")

File: modul 5/PRAK501-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>
int MaxBilangan(int a, int b, int c, int d){
    int maxbil = a;
    if(b > maxbil) maxbil = b;
    if(c > maxbil) maxbil = c;
    if(d > maxbil) maxbil = d;
    return maxbil;}
    
int main(){
    int a,b,c,d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    return 0;
}